token.go

```diff
@@ -0,0 +1,22 @@
+package auth
+
+import (
+	"github.com/go-martini/martini"
+	"net/http"
+)
+
+// AuthToken is the authorization token extracted from the request.
+type AuthToken string
```

Comment:
Maybe `Token` would be enough, since this is already namespaced by `package auth`.

It reads better to ask for an `auth.Token` rather than an `auth.AuthToken`.